plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/schema/confluent/TestConfluentSchemaRegistryTableDescriptionSupplier.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka.schema.confluent;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import io.airlift.units.Duration;
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.trino.plugin.kafka.KafkaTopicDescription;
+import io.trino.plugin.kafka.KafkaTopicFieldDescription;
+import io.trino.plugin.kafka.KafkaTopicFieldGroup;
+import io.trino.plugin.kafka.schema.TableDescriptionSupplier;
+import io.trino.spi.connector.SchemaTableName;
+import io.trino.spi.type.TestingTypeManager;
+import io.trino.spi.type.Type;
+import io.trino.testing.TestingConnectorSession;
+import org.apache.avro.Schema;
+import org.apache.avro.SchemaBuilder;
+import org.testng.annotations.Test;
+
+import java.util.List;
+import java.util.Optional;
+
+import static io.trino.spi.type.IntegerType.INTEGER;
+import static io.trino.spi.type.VarcharType.createUnboundedVarcharType;
+import static java.lang.String.format;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertTrue;
+
+public class TestConfluentSchemaRegistryTableDescriptionSupplier
+{
+    private static final String DEFAULT_NAME = "tests";
+    private static final SchemaRegistryClient SCHEMA_REGISTRY_CLIENT = new MockSchemaRegistryClient();
+
+    @Test
+    public void testTopicDescription()
+            throws Exception
+    {
+        TableDescriptionSupplier tableDescriptionSupplier = createTableDescriptionSupplier();
+        String topicName = "simple_topic";
+        SchemaTableName schemaTableName = new SchemaTableName(DEFAULT_NAME, topicName);
+        String subject = topicName + "-value";
+        SCHEMA_REGISTRY_CLIENT.register(subject, getAvroSchema(schemaTableName.getTableName(), ""));
+
+        assertTrue(tableDescriptionSupplier.listTables().contains(schemaTableName));
+
+        assertThat(getKafkaTopicDescription(tableDescriptionSupplier, schemaTableName)).isEqualTo(
```

Comment:
break line before `.isEqualTo`, thanks to that it looks like
```
assert actual
     condition expected
     condition expected
     condition expected
```    

It is much easier to read then.